discover.js

```diff
@@ -0,0 +1,166 @@
+#!/usr/bin/env node
+import { initializeDatabase, insertMeeting, getMeeting } from './db/init.js';
+import { createQueue, QUEUE_NAMES } from './queue/config.js';
+import 'dotenv/config';
+
+const BASE_URL = 'https://pub-cityofgainesville.escribemeetings.com';
+const API_URL = `${BASE_URL}/MeetingsCalendarView.aspx/GetCalendarMeetings`;
+
+function getDateRange(startDate, endDate) {
+  if (startDate && endDate) {
+    return {
+      start: startDate,
+      end: endDate
+    };
+  }
+
+  const now = new Date();
+  const start = new Date(now.getFullYear(), now.getMonth(), 1);
+  const end = new Date(now.getFullYear(), now.getMonth() + 1, 1);
+
+  const toISOStringWithOffset = (date) =>
+    date.toISOString().replace('Z', '-04:00');
+
+  return {
+    start: toISOStringWithOffset(start),
+    end: toISOStringWithOffset(end),
+  };
+}
+
+async function fetchMeetingsWithVideo(startDate, endDate) {
+  const { start, end } = getDateRange(startDate, endDate);
+  console.log(JSON.stringify({ 
+    message: 'Fetching meetings', 
+    start, 
+    end,
+    step: 'discovery'
+  }));
+
+  const res = await fetch(API_URL, {
+    method: 'POST',
+    headers: { 'Content-Type': 'application/json' },
+    body: JSON.stringify({
+      calendarStartDate: start,
+      calendarEndDate: end,
+    }),
+  });
+
+  const data = await res.json();
+  const meetings = data.d;
+
+  const meetingsWithVideo = meetings
+    .filter((meeting) => meeting.HasVideo)
+    .map((meeting) => ({
+      id: meeting.ID,
+      title: meeting.MeetingName,
+      meeting_url: `${BASE_URL}/Meeting.aspx?Id=${meeting.ID}`,
+      date: meeting.StartDate,
+      has_video: true
+    }));
+
+  console.log(JSON.stringify({
+    message: 'Found meetings with video',
+    count: meetingsWithVideo.length,
+    step: 'discovery'
+  }));
+  
+  return meetingsWithVideo;
+}
+
+export async function runDiscovery(options = {}) {
+  const { 
+    startDate = null, 
+    endDate = null, 
+    enqueueOnly = false 
```

Comment:
I don't know that discover needs to know about the queue at all

And enqueueOnly doesnt make sense as a name.